### PR TITLE
resolves #620.

### LINKED ISSUE #620: Retry Action for failed downloads.
The snackbars for "Download failed/couldn't refresh" or similar events should contain a "retry" action so user can retry easily and get improved experience.

### PR DESCRIPTION
**Pull request by gogeta95**
Added retry action to all the snackbars where downloads are failing.
resolves #620

### CODE REVIEW

File: android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java [hunk 1]
```diff
@@ -786,21 +806,4 @@ public void run() {
         });
     }
 
-    public static void getDaysBetweenDates(Date startdate, Date enddate) {
```
Comment (mananwason) [inline android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java]: Why is this method rearranged?

**Comment by mananwason** [comment]
@gogeta95 There is just one issue more after that this is good to go 👍

File: android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java (hunk 1 above)
Comment (gogeta95) [inline android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java]: it was done by android studio. according to standards, static methods go on top.

Comment (mananwason) [inline android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java]: okay didn't notice that.

**Comment by mananwason** [comment]
LGTM merging.

**Pull request merged**